build.sh

```diff
@@ -4,16 +4,20 @@ cd "$(dirname "$0")"
 
 # Available variables
 # -------------------
-declare PS_VERSION;      # -- PrestaShop version, defaults to latest
-declare PHP_VERSION;     # -- PHP version, defaults to recommended version for PrestaShop
-declare OS_FLAVOUR;      # -- either "alpine" (default) or "debian"
-declare SERVER_FLAVOUR;  # -- not implemented, either "nginx" (default) or "apache"
-declare TARGET_PLATFORM; # -- a comma separated list of target platforms (defaults to "linux/amd64")
-declare PLATFORM;        # -- alias for $TARGET_PLATFORM
-declare TARGET_IMAGE;    # -- docker image name, defaults to "prestashop/prestashop-flashlight"
-declare PUSH;            # -- set it to "true" if you want to push the resulting image
-declare ZIP_SOURCE;      # -- the zip to unpack in flashlight
-declare DRY_RUN;         # -- if used, won't really build the image. Useful to check tags compliance
+declare PS_VERSION;        # -- PrestaShop version, defaults to latest
+declare PHP_VERSION;       # -- PHP version, defaults to recommended version for PrestaShop
+declare OS_FLAVOUR;        # -- either "alpine" (default) or "debian"
+declare SERVER_FLAVOUR;    # -- not implemented, either "nginx" (default) or "apache"
+declare TARGET_PLATFORM;   # -- a comma separated list of target platforms (defaults to "linux/amd64")
+declare PLATFORM;          # -- alias for $TARGET_PLATFORM
+declare TARGET_IMAGE;      # -- docker image name, defaults to "prestashop/prestashop-flashlight"
+declare PUSH;              # -- set it to "true" if you want to push the resulting image
+declare ZIP_SOURCE;        # -- the zip to unpack in flashlight
+declare PRE_INSTALLED_MODULES;   # -- install modules during zip installation
```

Comment:
I don't think this should come along with the build time. It's rather a variable to be declared on top of `assets/run.sh`